src/charm.py

```diff
@@ -198,6 +216,68 @@ def _get_hydra_endpoint_info(self) -> Optional[str]:
 
         return oauth2_provider_url
 
+    def _get_juju_config_identity_schema_config_(self) -> Optional[Tuple[str, Dict]]:
```

Comment:
Why have those methods return tuples instead of keeping them scoped to 1 juju config?